src/components/academies/GoToAcademy.jsx

```diff
@@ -0,0 +1,63 @@
+import { Link } from 'react-router-dom';
+import { Alert, Button, Skeleton } from '@openedx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Error } from '@openedx/paragon/icons';
+import React from 'react';
+import useEnterpriseCustomer from '../app/data/hooks/useEnterpriseCustomer';
+import { useAcademies } from '../app/data';
+
+const GoToAcademy = () => {
+  const { data: academies, isError: fetchError, isLoading } = useAcademies();
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
+
+  if (isLoading) {
+    return (
+      <>
+        <div className="sr-only">Loading...</div>
+        <Skeleton height={100} />
+      </>
+    );
+  }
+
+  if (fetchError || academies?.length !== 1) {
+    return (
+      <Alert
+        variant="danger"
+        dismissible={false}
+        icon={Error}
+        open
```

Comment:
[suggestion] If you do keep the `Alert` messaging in here, I would recommend _always_ rendering the `Alert` but passing `open={false}` when it shouldn't be visible in the UI; rendering `Alert` initially with `open={true}` prevents the CSS animation from working when the `Alert` appears.